### PR TITLE
BigQuery Client integration with test case

### PR DESCRIPTION
**Pull request by ksahilCF**
Test Case should pass completely. No of records are fetched from BQ client

### CODE REVIEW

File: src/main/java/utils/GcpClient.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        ;
+        String tableName = SeleniumHelper.readParameters(table);
+        String query = "DROP TABLE `"
+                + projectId
+                + "."
+                + datasetName
+                + "."
+                + tableName
+                + "`";
+
```
Comment (tivv) [inline src/main/java/utils/GcpClient.java:37]: Do you want to run the query?

Comment (ksahilCF) [reply src/main/java/utils/GcpClient.java:37]: yes, i want to get the count of records transferred

Comment (tivv) [reply src/main/java/utils/GcpClient.java:37]: So I don't see where you run the query. I see only where you make the query string

Comment (ksahilCF) [reply src/main/java/utils/GcpClient.java:37]: just check the return statement it is calling the query function executing the string

Comment (tivv) [reply src/main/java/utils/GcpClient.java:37]: There is no return statement in this method

**Pull request closed without merging**